src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -236,15 +236,15 @@ static std::vector<SenderThreadParams*>* buildSenderParamsCustom
     {
       if (macroSubstitute(&payload, httpInfo.payload, ce) == false)
       {
-        LM_W(("Bad Input (not sending NotifyContextRequest: payload too large)"));
+        LM_W(("Runtime Error (not sending NotifyContextRequest: payload too large)"));
```

Comment:
In the case of missing ending '}', it is not payload too large ...
Perhaps macroSubstitute() should return enum values that describe the cause of the error ...
Actually (IMHO), a missing ending '}' should *not* cause an error, simply not being replaced and kept as is ...